src/components/tab-title/tab-title.tsx

```diff
@@ -228,13 +231,15 @@ export class TabTitle implements InteractiveComponent {
         e.preventDefault();
         break;
       case "ArrowRight":
+        e.preventDefault();
```

Comment:
Added these so horizontal scrolling does not occur when arrow keys were pressed.